test/dsets.c

```diff
@@ -990,14 +990,16 @@ test_compact_io(hid_t fapl)
 
             /* Verify the dataset's layout and fill message versions */
             if (fp->shared->low_bound == H5F_LIBVER_EARLIEST) {
-                VERIFY(dsetp->shared->layout.version, H5O_LAYOUT_VERSION_DEFAULT, "layout_ver_bounds");
```

Comment:
Replace usages of the `VERIFY` macro with equivalent logic for some test files that weren't really integrated with the testframe.c framework